e2e-tests/fixtures/Sequence.ts

```diff
@@ -15,15 +15,20 @@ export class Sequence {
   parcel: Locator;
   sequenceName: string;
   sequenceNameTextbox: Locator;
-  table: Locator;
-  tableRow: Locator;
-  tableRowDeleteButton: Locator;
+  sequenceTable: Locator;
+  sequenceTableRow: Locator;
+  sequenceTableRowDeleteButton: Locator;
+  workspaceModal: Locator;
+  workspaceName: string;
+  workspaceNameTextbox: Locator;
+  workspaceTable: Locator;
+  workspaceTableRow: Locator;
 
   constructor(public page: Page) {}
 
   async createSequence(parcelName: string): Promise<string> {
     await this.page.locator('button:has-text("New")').click();
-    await this.page.waitForURL('/sequencing/new');
+    await this.page.waitForURL('/sequencing/new**');
```

Comment:
Can this be a little more specific so that we can glean from it what to expect just by looking at it?